docs/en/building/basic-usage.md

```diff
@@ -0,0 +1,118 @@
+
+A new Dojo application will have example unit and function tests under the `tests` directory. Scripts for running tests are available in the `package.json` file.
+
+The fastest way to run tests is by running them in node using [@dojo/cli-test-intern](https://github.com/dojo/cli-test-intern)'s JIT builder.
+
+> Command line
+
+```bash
+npm test
+```
+
+This command uses [ts-node](https://www.npmjs.com/package/ts-node) to automatically build and execute unit tests located in `tests/unit` and build code from `src` on demand.
+
+> Command line
+
+```bash
+npm run test:unit
+```
+
+This command will build the Dojo application and tests into `dojo\test\unit` and use Intern to run tests locally using node and Chrome.
+
+> Command line
+
+```bash
+npm run test:functional
+```
+
+This command will build the Dojo application and functional tests into `dojo\test\functional` and use Intern to execute tests in Chrome using Selenium.
+
+Intern comes with support for running tests remotely on [BrowserStack](https://www.browserstack.com), [SauceLabs](https://saucelabs.com/), and [TestingBot](https://testingbot.com/). You may use one of these services by signing up for an account and providing your credentials to cli-test-intern. By default, all of the testing services will run tests against IE11, Firefox, and Chrome.
+
+More detailed information is available in the [@dojo/cli-test-intern](https://github.com/dojo/cli-test-intern) repository and from [Intern](https://theintern.io/).
+
+## Browser Support
+
+Dojo is an evergreen framework. By default the build will support the last two versions of the latest browsers when it was released. Any polyfills needed by Dojo to normalize functionality across browsers is conditionally provided by the `@dojo/framework/shim`. To support IE11 run the build with the `--legacy` flag.
+
+# Dojo Configuration
+
+Additional configuration options can be added in `.dojorc`. The options generally expand on the settings available through the command line and allow for more advanced features such as internationalization, code splitting, PWA manifests, and eliding code.
+
+`.dojorc` contains a JSON object with configuration information for any of the commands that are run via the dojo command line tool. Each command is allocated a section within the configuration object where its settings can be stored.
+
+```json
+{
+	"build-app": {
+		"pwa": {
+			"manifest": {
+				"name": "My Application",
+				"description": "My amazing application"
+			}
+		}
+	},
+	"test-intern": {},
+	"create-widget": {
+		"tests": "tests/unit"
+	}
+}
+```
+
+In this example we have a section for each of the three CLI command modules: [@dojo/cli-build-app](https://github.com/dojo/cli-build-app/), [@dojo/cli-test-intern](https://github.com/dojo/cli-test-intern), and [@dojo/cli-create-widget](https://github.com/dojo/cli-create-widget). Configuration is _always_ hierarchical and in the order of command => feature => configuration.
```

Comment:
Minor style consistency point - the new guides are favoring writing from a third-person perspective (if needed at all), essentially avoiding first-/second-person descriptions such as 'we'/'you'/'your'.

Reply:
I'm concerned using passive voice in tutorials and guides is less direct and engaging. I understand passive voice in customer contacts as a way to avoid confrontation and blame. Can we talk about this when we get a moment?